spec/neo4j-core/unit/query_spec.rb

```diff
@@ -639,6 +639,10 @@ def self.it_generates(cypher, params = {})
     describe '.create(q: {Person: {age: nil, height: 70}})' do
       it_generates 'CREATE (q:`Person` {age: {q_Person_age}, height: {q_Person_height}})', q_Person_age: nil, q_Person_height: 70
     end
+
+    describe '.create(q: {\'Child:Person\': {age: 41, height: 70}})' do
+      it_generates 'CREATE (q:`Child:Person` {age: {q_Child_Person_age}, height: {q_Child_Person_height}})', q_Child_Person_age: 41, q_Child_Person_height: 70
```

Comment:
Just to be sure, did you check that this creates `Child` and `Person` as two separate labels and not some mutant `Child:Person`?